src/timeseriesinsights/README.md

```diff
@@ -1,95 +1,129 @@
-Microsoft Azure CLI 'timeseriesinsights' Extension
-==========================================
+# Azure CLI timeseriesinsights Extension #
+This is the extension for timeseriesinsights
 
-This package is for the 'timeseriesinsights' extension:
-
-```sh
-az timeseriesinsights
+### How to use ###
+Install this extension using the below CLI command
 ```
-
-## Install
-
-```sh
 az extension add --name timeseriesinsights
 ```
 
-## Uninstall
-
-```sh
-az extension remove --name timeseriesinsights
+### Included Features ###
+#### timeseriesinsights environment ####
+##### Gen1 Create #####
 ```
-
-## Examples
-
-All commands are for Bash. For PowerShell, please set the variables like 
-
-```powershell
-$rg='rg1'
+az timeseriesinsights environment gen1 create --name "env1" --location westus --data-retention-time "P31D" \
```

Comment:
`\` is only supported by Linux Bash, we usually don't include `\` in examples. 

However, you can add a comment to tell the user line breaks are added for legibility only, like

https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code

```
// Line breaks for legibility only

https://login.microsoftonline.com/{tenant}/oauth2/v2.0/authorize?
client_id=6731de76-14a6-49ae-97bc-6eba6914391e
&response_type=code
&redirect_uri=http%3A%2F%2Flocalhost%2Fmyapp%2F
&response_mode=query
&scope=openid%20offline_access%20https%3A%2F%2Fgraph.microsoft.com%2Fmail.read
&state=12345
&code_challenge=YTFjNjI1OWYzMzA3MTI4ZDY2Njg5M2RkNmVjNDE5YmEyZGRhOGYyM2IzNjdmZWFhMTQ1ODg3NDcxY2Nl
&code_challenge_method=S256
```